modules/sdk-coin-ada/src/lib/transaction.ts

```diff
@@ -15,9 +15,15 @@ export interface TransactionInput {
   transaction_index: number;
 }
 
+export interface AssetList {
+  policy_id: string;
+  asset_name: string;
+  quantity: string;
+}
 export interface TransactionOutput {
   address: string;
   amount: string;
+  multiAssets?: CardanoWasm.MultiAsset;
```

Comment:
Can we make this a non CardanoWasm type? Define our own `MultiAsset` interface?

Reply:
sure we can change that, but any specific reason why ? originally made it the CardanoWasm type because I'd have to extract out the fields anyway, have a new interface and then construct it back

Reply:
^can address this in a followup if needed